app/controllers/bulk_update/recommendations_uploads_controller.rb

```diff
@@ -26,6 +26,15 @@ def create
       end
     end
 
+    def download_errors
```

Comment:
I think this controller has to many endpoints. Ideally it should be restful.

I've created a new one for the `checks` method in #3162.

Maybe something like `BulkUpdate::RecommendationsErrorsController`?

Reply:
good suggestion! i was feeling the same then i just dumped it in 😂  i like recommendationserrors 👍